apps/studio/components/interfaces/Account/Preferences/AccountInformation.tsx

```diff
@@ -17,21 +27,76 @@ const AccountInformation = ({ profile }: { profile?: Profile }) => {
       }
     >
       <Panel.Content>
-        <div className="space-y-2">
-          <Input
-            readOnly
-            disabled
-            label="Username"
-            layout="horizontal"
-            value={profile?.username ?? ''}
-          />
-          <Input
-            readOnly
-            disabled
-            label="Email"
-            layout="horizontal"
-            value={profile?.primary_email ?? ''}
-          />
+        <div className="grid gap-4">
+          <div className="grid grid-cols-12 gap-6">
+            <Label_Shadcn_ className="col-span-4 text-foreground-light">Username</Label_Shadcn_>
+            <Input_Shadcn_
+              className="col-span-8"
+              readOnly
+              disabled
+              placeholder="Username"
+              value={profile?.username ?? ''}
+            />
+          </div>
+          <div className="grid grid-cols-12 gap-6">
+            <Label_Shadcn_ className="col-span-4 text-foreground-light">Email</Label_Shadcn_>
+
+            <div className="col-span-8">
+              <Input_Shadcn_
+                className="w-full"
+                readOnly
+                disabled
+                placeholder="Email"
+                value={profile?.primary_email ?? ''}
+              />
+              <div className="col-span-8 mt-4">
+                <Collapsible_Shadcn_ className="-mt-1.5 pb-1.5">
+                  <CollapsibleTrigger_Shadcn_ className="group  font-normal p-0 [&[data-state=open]>div>svg]:!-rotate-180">
+                    <div className="flex items-center gap-x-2 w-full">
+                      <p className="text-xs text-foreground-light group-hover:text-foreground transition">
+                        Can I change my email?
+                      </p>
+                      <ChevronDown
+                        className="transition-transform duration-200"
+                        strokeWidth={1.5}
+                        size={14}
+                      />
+                    </div>
+                  </CollapsibleTrigger_Shadcn_>
+                  <CollapsibleContent_Shadcn_ className=" mt-2">
+                    <div className="bg-surface-200 p-4 rounded-lg grid gap-4">
+                      <div>
+                        <h4 className="text-sm">If you used GitHub to login: </h4>
```

Comment:
We should make this smarter by identifying the type of the session and showing the specific instructions like we do on line 100:

```     
     {session?.user.app_metadata.provider === 'email' && (
```